Tests/featureCollectionTest.js

```diff
@@ -0,0 +1,18 @@
+var homePage = require('../Pages/homePage.js'),
+    
+    fashionCategoryPage = require ("../Pages/fashionCategoryPage"),
+    
+    data = require("../Data/data.js");
+    
+browser.waitForAngularEnabled(false);
+browser.ignoreSynchronization = true;
+browser.manage().window().maximize();
+
+describe("012: Feature Collection", function(){
+    it("001: User is able to open item from 'Feature Collection' section", function(){
+        homePage.openPageURL(data.homepageLink)
```

Comment:
I combined them into one method, in some tests I didn't write .then(() => homePage.waitForLoginButton()) part, so now test will be more stable, because whenever home page of the page is opened, it wil wait for full loaded HTML